jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss

```diff
@@ -32,15 +32,26 @@
         @extend %floating-panel;
         display: inline-block;
         margin-left: 10px;
+
+        .button--inline-deployment {
+            padding-left: 10px;
+            padding-right: 10px;
+
+            &:first-child {
+                border-top-right-radius: 0;
+                border-bottom-right-radius: 0;
+            }
+
+            &:last-child {
+                border-top-left-radius: 0;
+                border-bottom-left-radius: 0;
+            }
+        }
     }
 
-    &__install-button {
-        @extend %fade-in;
-        @extend %floating-panel;
-        display: inline-block;
-        margin-left: 10px;
-        padding: 10px;
-        color: $cool-grey;
+    .button--inline-neutral {
```

Comment:
Typically we have tried to have the styles for the various components within that components folder with modifiers for the various wrappers that it would be put in. Which I think is what you're doing in the _generic-button styles. However cases like this, and elsewhere in the code, you're providing the deployment-bar styles in the _generic-button styles but then also modifying some of them here. I feel like all of these little modifications should be moved into the _generic-button style file under the associated component wrapper.